src/tools/dart-run.md

```diff
@@ -129,13 +129,30 @@ $ dart run foo arg1 arg2
 ## Debugging
 
 To enable debugging, pass one or more debugging options.
```

Comment:
**suggestion:** I would show all of the arguments together to help illustrate possibilities, then explain what each does.

```terminal
$ dart run \
   [--enable-asserts FILEPATH] \
   [--observe] \
   [--pause-isolates-on-start]
```

This is consistent with the [Google Style Guide](https://developers.google.com/style/code-syntax#optional-arguments).

Reply:
If we want to go this direction, it seems more like a complete reference (rather than a small debugging tutorial) and we would maybe need to identify each debug flag, rather than just a few. Is that something we should do for this section?

Reply:
**comment:** This section does call out three specific flags. It would help to see them together in context. That said, a slight change the first sentence can limit your scope.
```suggestion
To enable debugging, add one or more of these common debugging options to your `dart run` command:
```